sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/clangtidy/CxxClangTidySensor.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Sonar C++ Plugin (Community)
+ * Copyright (C) 2010-2017 SonarOpenCommunity
+ * http://github.com/SonarOpenCommunity/sonar-cxx
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.plugins.cxx.clangtidy;
+
+import java.io.File;
+import java.util.Scanner;
+import java.util.regex.MatchResult;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.sonar.api.batch.sensor.SensorContext;
+import org.sonar.api.batch.sensor.SensorDescriptor;
+import org.sonar.api.config.Settings;
+import org.sonar.api.utils.log.Logger;
+import org.sonar.api.utils.log.Loggers;
+import org.sonar.plugins.cxx.CxxLanguage;
+import org.sonar.plugins.cxx.utils.CxxMetrics;
+import org.sonar.plugins.cxx.utils.CxxReportSensor;
+
+/**
+ * Sensor for clang-tidy
+ */
+public class CxxClangTidySensor extends CxxReportSensor {
+  public static final Logger LOG = Loggers.get(CxxClangTidySensor.class);
+  public static final String REPORT_PATH_KEY = "sonar.cxx.clangtidy.reportPath";
```

Comment:
i think the output cannot be change, at least did not find any reference to that in the command line. So its basically its hard code to the regex that is define below